kv/txn_correctness_test.go

```diff
@@ -911,17 +936,6 @@ func TestTxnDBPhantomDeleteAnomaly(t *testing.T) {
 // history above) and may set A=1, B=1.
 func TestTxnDBWriteSkewAnomaly(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	txn1 := "SC(A-C) W(A,A+B+1) C"
-	txn2 := "SC(A-C) W(B,A+B+1) C"
-	verify := &verifier{
-		history: "R(A) R(B)",
-		checkFn: func(env map[string]int64) error {
-			if !((env["A"] == 1 && env["B"] == 2) || (env["A"] == 2 && env["B"] == 1)) {
-				return util.Errorf("expected either A=1, B=2 -or- A=2, B=1, but have A=%d, B=%d", env["A"], env["B"])
-			}
-			return nil
-		},
-	}
-	checkConcurrency("write skew", onlySerializable, []string{txn1, txn2}, verify, true, t)
-	checkConcurrency("write skew", onlySnapshot, []string{txn1, txn2}, verify, false, t)
+	runWriteSkewTest(t, roachpb.SERIALIZABLE)
```

Comment:
I don't have the isolation level available in checkFn though, and I doubt that plumbing it there just for that is a better solution.